docs/src/modules/components/ApiPage.js

```diff
@@ -3,11 +3,16 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
 import { exactProp } from '@material-ui/utils';
+import { styled } from '@material-ui/core/styles';
+import Tooltip from '@material-ui/core/Tooltip';
 import Typography from '@material-ui/core/Typography';
 import { useTranslate, useUserLanguage } from 'docs/src/modules/utils/i18n';
 import HighlightedCode from 'docs/src/modules/components/HighlightedCode';
 import MarkdownElement from 'docs/src/modules/components/MarkdownElement';
 import AppLayoutDocs from 'docs/src/modules/components/AppLayoutDocs';
+import InfoOutlined from '@material-ui/icons/InfoOutlined';
+
+const Asterisk = styled('abbr')(({ theme }) => ({ color: theme.palette.error.main }));
```

Comment:
It would be nice if we expose something like `<styled.abbr />` so that I can use `sx` without creating component every time. @oliviertassinari @mnajdova

Reply:
Do you mean #23220?